XRTK-Core/Packages/com.xrtk.core/Services/MixedRealityToolkit.cs

```diff
@@ -131,6 +131,7 @@ public void ResetConfiguration(MixedRealityToolkitConfigurationProfile profile)
 
         #region Mixed Reality runtime service registry
 
+        // ReSharper disable once InconsistentNaming
```

Comment:
I don't like these Resharper comments here. Not a tool everyone uses. But I can live with it if you guys kept them so far.

Reply:
Good thing they're just comments 😉